cmake/FindFVAD.cmake

```diff
@@ -0,0 +1,29 @@
+find_path(FVAD_INCLUDE_DIR
```

Comment:
Maybe someone else has more cmake knowledge, but I suggest to start with pkg-config. Compare with FindAAC.cmake!

```cmake
find_package(PkgConfig QUIET)
pkg_search_module(AAC fdk-aac)
...
```

Reply:
I'll take a look.

Reply:
Done